_includes/style.scss

```
@@ -156,48 +176,27 @@ h3 {
     background-image: url('/icons/light-theme.svg');
 }
 
-
 /** theme specific css **/
 @media (prefers-color-scheme: dark) {
-    body {
-        background-color: #000103;
-        color: white;
-    }
-
-    a {
-        color: #ff0080;
-    }
-
-    .resource {
-        border-color: white;
+    :root:not(.light) {
+        $background-color: black;
+        $font-color: white;
+
+        body {
+            background-color: $background-color;
+            color: $font-color;
+        }
+
+        .resource {
+            border-color: $font-color;
+        }
+
+        .phone-icon {
+            background-image: url('icons/phone-light.svg');
```

Should these URLs be absolute, just in case?
```suggestion
            background-image: url('/icons/phone-light.svg');
```

---

This is outdated now, not using urls